### DISCUSSION TITLE
I want to run a customize script which can run a select menu how to make it?

### DISCUSSION THREAD

**Discussion by OriLiMu** [sxyazi/yazi#1404] [1. Q&A] [answered]
I want to mv the current file to the target folders. 
So I need a keymap let's say `mm` to trigger it. Then a menu shows with target folders, I can select the one I wanted to move the current file to. 
How to make it?

**Comment by sxyazi** [comment, marked as answer]
Here is an example https://github.com/BBOOXX/file-actions.yazi
